test/Benchmarks/src/Text/Regex_Bench.enso

```diff
@@ -0,0 +1,84 @@
+from Standard.Base import all
+
+import Standard.Base.Data.Text.Regex.Internal.Replacer.Replacer
+
+from Standard.Test import Bench, Faker
+
+
+type Data
+    Value ~two_nums ~just_two_nums ~double_two_nums ~a_regex ~regexes ~replacers ~one_num_regex ~two_nums_regex
+
+    create =
+        faker = Faker.new
+        a_regex = Regex.compile "(\d)ab(\d)"
+        Data.Value (build_two_nums faker) (build_just_two_nums faker) (build_double_two_nums faker) a_regex (build_regexes faker) (build_replacers faker) (Regex.compile "(\d+)") (Regex.compile "(\d+)[a-z]+(\d+)")
+
+list_count = 1000000
+regex_count = 10000
+
+# "ab12cdefg634hi"
+build_two_nums faker = 0.up_to list_count . map _->
+    (faker.alpha 2) + (faker.integer 0 100).to_text + (faker.alpha 8) + (faker.integer 0 100).to_text + (faker.alpha 2)
+
+# "12cdefg634"
+build_just_two_nums faker = 0.up_to list_count . map _->
+    (faker.integer 0 100).to_text + (faker.alpha 8) + (faker.integer 0 100).to_text
+
+# "ab12cdefg634hiab12cdefg634hi"
+build_double_two_nums faker =
```

Comment:
Should this be called `build_four_nums`? 😅

Reply:
Yes "four" is a better way of writing "double_two", changed. 😃